microovn/.golangci.yml

```diff
@@ -1,3 +1,8 @@
+issues:
+  exclude-use-default: false
+  exclude-rules:
+    - linters: [errcheck]
+      text: "Error return value of .((os\\.)?std(out|err)\\..*|.*Close|.*Flush|os\\.Remove(All)?|.*printf?|os\\.(Un)?Setenv). is not checked"
```

Comment:
Could we have a short comment here explaining what's the purpose of this `exclude-rules` statement?

Reply:
Yes, it definitively deserves one, on it.

The gist of it is that the default exclude rules disables the GoDoc requirement check, so we need to disable the default exclude rules, which then also disables the one we add back.